ecl/network/v2/fic_gateways/testing/request_test.go

```diff
@@ -0,0 +1,69 @@
+package testing
+
+import (
+	"fmt"
+
+	"github.com/nttcom/eclcloud/ecl/network/v2/fic_gateways"
+	"github.com/nttcom/eclcloud/pagination"
+
+	"net/http"
+	"testing"
+
+	fake "github.com/nttcom/eclcloud/ecl/network/v2/common"
+	th "github.com/nttcom/eclcloud/testhelper"
+)
+
+func TestList(t *testing.T) {
```

Comment:
I think this method name should be added suffix like "FICGateway". The method name "TestList" is output to the test log and we cannot recognize the test target.